telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/Chat.java

```diff
@@ -30,7 +30,8 @@
 @AllArgsConstructor
 public class Chat implements BotApiObject {
 
-    private static final String ID_FIELD = "id";
+    // Variable name changed from CHAT_FIELD to CHAT_ID_FIELD for refactoring
+    private static final String CHAT_ID_FIELD = "id";
```

Comment:
In Telegram api, the Chat object has the "id" field, not "chat_id". It would be convenient to leave it as is